Dockerfile

```diff
@@ -4,6 +4,10 @@ USER root
 
 ENV ANDROID_SDK_ROOT=/opt/android-sdk
 
+RUN apt-get update && apt-get install -y ruby ruby-dev ruby-bundler build-essential \

```

Comment:
Why install `ruby-bundler` if you install the gem individually next?

Reply:
The intention here was to pre-load the container with fastlane so that it didn't get re-downloaded with every build. The android release pipeline runs `bundle install` regardless, just in-case a newer version is suddenly required before the container can be updated. I might remove that bit though, since I just saw now that the `bundle install` failed my build (due to having insufficient permissions to update dependencies).

Reply:
I've removed the installation of bundler because it wasn't working very well for me within the container. I don't think that there will be a significant need to update the fastlane version very often. It's probably best to use the [`min_fastlane_version`](https://docs.fastlane.tools/actions/min_fastlane_version/) fastlane action to fail the build instead if we need to enforce using a later version.